package.json

```
@@ -11,7 +11,8 @@
   },
   "scripts": {
     "lint": "eslint --ext js && tslint 'src/**/*.ts?(x)'",
```

@Ceshion I recommend sticking with eslint instead of tslint. eslint supports js/jsx and ts/tsx. Also glad we are using typescript :D 

This line can be updated to... 
```suggestion
    "lint": "eslint --ext ts,tsx src && tsc --noEmit",
```

And `tslint.json` could be removed.

---

tsc emits strict typing errors from within node_modules, so in addition to globbing these I'm also disabling libCheck-- we still want strict type checks within our code, and one of these such errors is from React itself. given that, I don't think there's any notion of i.e. removing the libraries giving these errors

---

I’m not sure I followed, but I agree skipLibCheck can be true in tsconfig.json.

Here’s an example config I use in another project that uses react, typescript, eslint and prettier - https://github.com/avimoondra/fishbowl/blob/master/app/tsconfig.json#L10

---

(Typing from mobile btw)